crates/wasm-smith/src/core.rs

```diff
@@ -2549,6 +2635,16 @@ impl Module {
             }
         }
     }
+
+    fn arbitrary_shared(&self, u: &mut Unstructured) -> Result<bool> {
+        Ok(self.config.shared_everything_threads_enabled && u.arbitrary()?)
```

Comment:
We might want to dial down the frequency of shared things compared to unshared ones? This would be a good place to do that.

Reply:
Yeah I think that'd be reasonable. Something like `u.ratio(1, 10)?` or similar perhaps? 

This might also be a good place to use the field I mentioned above on `Self` to force shared-ness since that could be used to make this method always return `true` for example.